app/views/organisations/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "layouts/form_errors", resource: @organisation %>
```

Comment:
I'd recommend the use of https://design-system.service.gov.uk/components/error-message/ to help out users more here. It seems like a minor improvement, but I think it's better to address this first time round rather than launch something that deviates from GDS standards.

Tom mentioned this, but it's kind of got lost in the thread.

Reply:
Specifically, the 
> Show an error message next to the field 
advice.

You can see how other parts of the app use the mentioned classes (like `govuk-input--error`) if it's not clear how to achieve this.